modules/module-headings.php

```diff
@@ -1,229 +1,229 @@
-<?php
-// Do not edit this file. It's generated by jetpack/tools/build-module-headings-translations.php
-
-/**
- * For a given module, return an array with translated name, description and recommended description.
- *
- * @param string $key Module file name without .php
- *
- * @return array
- */
-function jetpack_get_module_i18n( $key ) {
-	static $modules;
-	if ( ! isset( $modules ) ) {
+<?php
+// Do not edit this file. It's generated by jetpack/tools/build-module-headings-translations.php

```

Comment:
@jom yup! We'll take care of this later in the cycle.  You can go ahead and revert this file